modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+
+NOTE: The {productname} public API, DomParser uses, in turn, the same-named browser parsing API: https://developer.mozilla.org/en-US/docs/Web/API/DOMParser[DomParser].
+
+Also, the `media` plugin’s `media_scripts` option was deprecated in {productname} 5.10.
+
+// Based on the code it looks like `media_scripts` was meant to be a way to provide an allowed list of <script>s that could be used to embed media. However, somewhere along the lines it broke and no longer works.
+
+As of the release of {productname} 6.0, this option has been removed completely.
+
+[[noneditable]]
+==== Noneditable
+
+The `noneditable` plugin is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `noneditable` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[paste]]
+==== Paste
+
+The `paste` plugin’s functionality is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `paste` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+The `paste_data_images` option now defaults to `true`. When `paste` was a plugin, this option was, by default, set to `false`, which prevented images being pasted from the local machine.
+
+The `mceInsertClipboardContent` argument, `content`, has been renamed. It is now `html`.
+
+The new name is a more accurate reflection of what sort of data the argument passes.
+
+NOTE: `PastePreProcess` also takes an argument, `content`. This argument, for this command, has not changed.
+
+
+[[print]]
+==== Print
+
+`print` functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `print` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[table]]
+==== Table
+
+The `table` plugin’s functionality is, as of {productname} 6.0, entirely focussed on the user interface (UI) for creating and editing tables.
+
+The code that generates tables is now a Core part of {productname} 6.0.
+
+This, now core feature of {productname} 6.0, also includes the following fixes, changes, and improvements.
+
+* If the selected cells in the `tfoot` section of a table were header elements (ie were `th` elements), pressing the `table` row header toolbar button (which invokes the `mceTableRowType` command) returned the row as a header row.
```

Comment:
Should this be a list? It seems like it should just be a paragraph or something:
![image](https://user-images.githubusercontent.com/1575550/161907633-9bbdc854-ff48-4293-9292-03f2b8de5945.png)